### ISSUE TITLE
Jarify IJ1 tools and toolsets

### ISSUE THREAD

**Issue by tferr** [tferr/Scripts#24]
Discussed in the [forum](http://forum.imagej.net/t/install-bars-with-micro-manager/4524). The idea is to bundle all IJ1 macro tools (Action Tools, Menu Tools, etc.) and macro toolsets in the main jar file, so that it is easier to run them from MicroManager.

**Comment by tferr** [comment]
This may be actually hard to implement.

On one hand, Scijava does not seem to support IJ1 macro tools. E.g., this installs no tool in the IJ toolbar:
```groovy
// @ScriptService s
s.run("test.ijm", "macro 'Test Action Tool - C00bL1de0L1ee1' {showMessage('Tool not clicked');}", false, null);
```
On the other IJ1's MacroRunner is hardwired to load macro tools [exclusively from ij.jar](https://github.com/imagej/imagej1/blob/2a6c191b027b5b1f5f22484506159f80adda21c5/ij/plugin/MacroInstaller.java#L377-L393).

**Comment by tferr** [comment]
This is now implemented in v1.5.1
